winlogbeat/beater/winlogbeat.go

```diff
@@ -114,8 +106,8 @@ func (eb *Winlogbeat) Setup(b *beat.Beat) error {
 
 	// Create the event logs. This will validate the event log specific
 	// configuration.
-	eb.eventLogs = make([]eventlog.EventLog, 0, len(eb.config.Winlogbeat.EventLogs))
-	for _, config := range eb.config.Winlogbeat.EventLogs {
+	eb.eventLogs = make([]eventlog.EventLog, 0, len(config.EventLogs))
+	for _, config := range config.EventLogs {
 		eventLog, err := eventlog.New(config)
```

Comment:
moved eventlog.New into `init` method. Not sure about adding checkpoint to `init`. It checks the checkpoint file is available, can be parsed, can be written to + starts some worker. All of this required by `-configtest`?